docs/jwst/pathloss/description.rst

```diff
@@ -49,6 +49,58 @@ The form of the 2-D correction (point or uniform) that's appropriate for the
 data is divided into the SCI and ERR arrays and propagated into the variance
 arrays of the science data.
 
+The MSA reference file contains 2 entries: one for a 1x1 slit and one for a 1x3 slit.
+Each entry contains the pathloss correction for point source and uniform sources.
+The former depends on the position of the target in the fiducial shutter and
+wavelength, whereas the latter depends on wavelength only.  The point source 
+entry consists of a 3-d array, where 2 of the dimensions map to the location
+of the source (ranging from -0.5 to 0.5 in both X and Y), while the third dimension
+carries the wavelength dependence.  The 1x3 shutter is 3 times as large in Y as in X.
+
+The entry to use for a point source target is determined by looking at the shutter_state
+attribute of the slit used.  This is a string with a length equal to the number
+of shutters that make up the slit, with 1 denoting an open shutter, 0 a closed
+shutter and x the fiducial (target) shutter.  The reference entry is determined
+by how many shutters next to the fiducial shutter are open:
+
+ * if both adjacent shutters are closed, the 1x1 entry is used.  A matching
```

Comment:
aaaargh. Just looked at the resulting rtd pages and the formatting of this whole section that's supposed to use bullets (or whatever) is wonky. Can you try some kind of different formatting?

Reply:
See https://jwst-pipeline--8106.org.readthedocs.build/en/8106/jwst/pathloss/description.html#nirspec